data/users.ddl

```diff
@@ -12,5 +12,6 @@ create table users (
   geo varchar(32),
   company_name varchar(32),
   company_catchPhrase varchar(32),

```

Comment:
should we remove these company_* columns?  On the one hand they don't really anything, on the other hand those columns would be a pain to remove :-/